WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1501,28 +1501,7 @@ private void onUploadSuccess(MediaModel media) {
         }
     }
 
-    private void onUploadError(MediaModel media, MediaError error) {
-        String localMediaId = String.valueOf(media.getId());
-
-        Map<String, Object> properties = null;
```

Comment:
This code was doing multiple things 
1. Track failed upload
2. Construct error message
3. Determine MimeType/MediaType

However, as mentioned below - error and mediaType was never being used. So the only remaining thing is the tracking which was moved to MediaEditor.